smart-contracts/test/Unlock/Unlock.js

```diff
@@ -1,16 +1,19 @@
 process.env.NODE_ENV = 'test'
 
 const Unlock = artifacts.require('./Unlock.sol')
+const TestHelper = require('zos').TestHelper
 const shared = require('./behaviors/shared')
 
 contract('Unlock', (accounts) => {
+  const proxyAdmin = accounts[1]
   const unlockOwner = accounts[2]
   describe('Standard Unlock contract', function () {
     beforeEach(async function () {
-      this.unlock = await Unlock.new()
-      await this.unlock.initialize(unlockOwner)
+      const project = await TestHelper({ from: proxyAdmin })
+      const proxy = await project.createProxy(Unlock, { initMethod: 'initialize', initArgs: [unlockOwner], initFrom: unlockOwner })
+      this.unlock = await Unlock.at(proxy.address)
     })
-
-    shared.shouldBehaveLikeV1(accounts, unlockOwner)
+    let _logIndex = 0
```

Comment:
Could you document (via a comment) what that is for?

Reply:
you bet.